README.md

```diff
@@ -69,27 +70,270 @@ class Genre {
+		let last = this.author.lastName;
+		let date = this.publishingDate;
+		return `${title} was written by ${first} ${last} and published in ${date}.`;
+	}
+}
+```
+
+Here, we've created a `caption` getter that uses some properties of `Book`
+and some of `Author`. This is the most basic way classes can work together - by
+depending on each other for related data.
+
+## Recognize How Multiple Classes Can Share Behaviors
+
+In addition to data, classes can also utilize each other's methods. For
+instance, we could add a getter to our `Author` class called `fullname`
+that returns a `firstName` and `lastName` interpolated together:
+
+```js
+class Author {
+	constructor(firstName, lastName) {
+		this.firstName = firstName;
+		this.lastName = lastName;
+	}
+
+	get fullname() {
+		return `${this.firstName} ${this.lastName}`;
+	}
+}
+```
+
+We could then rewrite our `caption` getter to use `fullname` pseudo-property
+rather than the actual properties of `Author`:
+
+```js
+class Book {
+	constructor(title, author, genre, publishingDate) {
+		this.title = title;
+		this.author = author;
+		this.genre = genre;
+		this.publishingDate = publishingDate;
+	}
+
+	get caption() {
+		let title = this.title;
+		let name = this.author.fullname;
+		let date = this.publishingDate;
+		return `${title} was written by ${name} and published in ${date}.`;
+	}
+}
+
+lordOfTheRings.caption;
+// => 'The Lord of the Rings was written by J.R.R. Tolkien and published in 1954.'
+```
+
+Here, we're using some properties from `Book`, and `fullname` from `Author`.
+
+## Recognize How Classes Working Together Develop Dependencies
+
+One thing to always keep in mind when working with multiple classes:
+**If one class uses data or behaviors from another class, the first class
+becomes _dependent_ on the second.**
+
+In our `Book` class, for instance, we've written `caption` using
+`this.author.fullname`. The `caption` getter is expecting there to be a
+`fullname` property (or pseudo-property) present in `this.author`. If we
+mistakenly created a `Book` instance using only `String`s again and then tried
+to use `caption` we will have a problem:
+
+```js
+// A new book incorrectly created with string parameters for author and genre
+let unfinishedTales = new Book(
+	'Unfinished Tales',
+	'J.R.R. Tolkien',
+	'Fantasy',
+	'1980'
 );
+
+unfinishedTales;
+// => Book {
+//    title: 'Unfinished Tales',
+//    author: 'J.R.R. Tolkien',
+//    genre: 'Fantasy',
+//    publishingDate: '1954' }
+
+unfinishedTales.author;
+// => 'J.R.R. Tolkien'
+
+// unfinishedTales.author is just a String and has no `fullname` property
+unfinishedTales.author.fullname;
+// => undefined
+
+// when `fullname` is called from within `caption`, it produces undefined:
+unfinishedTales.caption;
+// => 'Unfinished Tales was written by undefined and published in 1980.'
 ```
 
-Now that we've encapsulated these related things, we can
+Because of how we have written `Book`, it has become dependent on `Author`. If
```

Comment:
Something like: "The way we originally set up `Book`, it is dependent on `Author`"....